src/Jackett.Common/Indexers/NewRealWorld.cs

```diff
@@ -168,15 +164,10 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
                 foreach (var row in rows)
                 {
-                    var release = new ReleaseInfo();
-                    release.MinimumRatio = 0.75;
-                    release.MinimumSeedTime = 0;
-
 
                     var qDetailsLink = row.QuerySelector("a[href^=details.php?id=]");
-                    release.Title = qDetailsLink.Text();
 
-                    if (!query.MatchQueryStringAND(release.Title))
+                    if (!query.MatchQueryStringAND(qDetailsLink.Text()))
```

Comment:
we are using this code below, just declare `var title = qDetailsLink.TextContent`